packages/tasks/src/tasks/image-text-to-image/data.ts

```diff
@@ -0,0 +1,54 @@
+import type { TaskDataCustom } from "../index.js";
+
+const taskData: TaskDataCustom = {
+	datasets: [],
+	demo: {
+		inputs: [
+			{
+				filename: "image-text-to-image-input.png",
```

Comment:
I think these should go here: https://huggingface.co/datasets/huggingfacejs/tasks/tree/main but I don't see the folder or a PR
same goes for other file too!